packages/deployment/scripts/deploy-gov-v2.ts

```
@@ -0,0 +1,221 @@
+            type: 'confirm',
+            name: 'confirmPeering',
+            message: kleur
+              .yellow()
+              .bold(
+                '\n⚠️  WARNING: This step requires all chains to be deployed first.\n' +
+                  'Make sure you have already run steps 1 & 2 on all networks you want to peer.\n' +
+                  'Are you sure all chains are ready for peering?',
+              ),
+            initial: false,
+          })
+          if (confirmPeering) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)